en-US/Docs/WhatsNew.md

```diff
@@ -27,33 +27,33 @@ ___
 
 Adafruit has [updated their starter pack](https://www.adafruit.com/product/2733) to now include Raspberry Pi 3 and Windows 10 IoT Core.
 
 
 ## New major features
 ___ 
-* Windows IoT Remote Client - remote into your IoT device to control and view what is displayed on your IoT device from your desktop or phone
-* Store integration - connect Windows 10 IoT Core to the store to service applications 
-* NOOBS - 8GB cards now supported 
-* Better Azure IoT Hub connectivity - provision your device with a device identity in the cloud 
+* [Windows IoT Remote Client]({{site.baseurl}}/{{page.lang}}/Docs/RemoteDisplay.htm) - remote into your IoT device to control and view what is displayed on your IoT device from your desktop or phone
+* [Store integration]({{site.baseurl}}/{{page.lang}}/Docs/Store.htm) - connect Windows 10 IoT Core to the store to service applications 
+* [NOOBS]({{site.baseurl}}/{{page.lang}}/Docs/GetStarted/noobs/getstartedstep2.htm) - 8GB cards now supported 
+* Better [Azure IoT Hub]({{site.baseurl}}/{{page.lang}}/Docs/cloudintro.htm) connectivity - provision your device with a device identity in the cloud 
 * [Shell capabilities](https://github.com/ms-iot/samples/tree/develop/IoTHomeAppSample) - multiple foreground app support
 * [IoTivity AllJoyn Device System Bridge](https://github.com/ms-iot/samples/tree/develop/AllJoyn/Samples/OICAdapter)
-* Arduino INO integration - work better with Arduino hardware with this command line toolkit
+* [Arduino INO integration](http://inotool.org/) - work better with Arduino hardware with this command line toolkit
 
-### Improved documentation
-* Updated samples page - new tagging system to sort through different samples 
-* Updated documentation page - added a table of contents (finally) and new documentation
+### [Improved documentation]({{site.baseurl}}/{{page.lang}}/Docs.htm)
+* Updated [samples page]({{site.baseurl}}/{{page.lang}}/Samples.htm) - new tagging system to sort through different samples 
+* Updated [documentation page]({{site.baseurl}}/{{page.lang}}/Docs.htm) - added a table of contents and new documentation
 
-### Getting started
-* New [getting started](https://developer.microsoft.com/en-us/windows/iot/GetStarted) flow  
-* Automatically set up your Wi-Fi connection through the IoT Dashboard
+### [Getting started]({{site.baseurl}}/{{page.lang}}/Explore/GetStarted.htm)
+* New [getting started]({{site.baseurl}}/{{page.lang}}/Explore/GetStarted.htm) flow  
+* Automatically set up your Wi-Fi connection through the [IoT Dashboard]({{site.baseurl}}/{{page.lang}}/Docs/IoTDashboard.htm)
  
-### For commercial builders
-* Retail signing with cross certs - cross sign your own drivers and be compatible with secure boot 
-* Trusted Platform Module (TPM) support - provision your azure key into a virtual or physical TPM 
+### For [commercial builders]({{site.baseurl}}/{{page.lang}}/Docs/CreateIoTCorePro.htm)
```

Comment:
This is one I think we should leave, since we don't provide that link in the bullets below